canopen_tests/config/canopen_system/bus.yml

```diff
@@ -0,0 +1,19 @@
+options:
```

Comment:
I assume you added this to make test independable?

Reply:
Yes, we had changed the bus.yaml for cia402 and it broke cia402_system launch. So I'd rather keep things seperate.